Src/BootCamp.Chapter/Program.cs

```diff
@@ -2,11 +2,99 @@
 
 namespace BootCamp.Chapter
 {
+    class Person
+    {
+        private string _name;
+        private string _age;
+        private string _weight;
+        private string _height;
+
+
+        public string Name {
+            get
+            {
+                return _name;
+            }
+            set
+            {
+                _name = value;
+            } 
+        
+        }
+
+        public string Age
+        {
+            get
+            {
+                return _age;
+            }
+            set
+            {
+                _age = value;
+            }
+
+        }
+
+        public string Weight {
+            get
+            {
+                return _weight;
+            }
+            set
+            {
+                _weight = value;
+            }
+        }
+
+        public string Height
+        {
+            get
+            {
+                return _height;
+            }
+            set
+            {
+                _height = value;
+            }
+
+        }
+
+        public Person(string name, string age, string weight, string height)
+        {
+            _name = name;
+            _age = age;
+            _weight = weight;
+            _height = height;
+        }
+
+        public double BMIcalc()
```

Comment:
BMICalc is a noun. Methods normally should be verbs. So a better name would be CalculateBmi. Also avoid using abreviations like that, it is indeed less code, but it reads worse.

Also, in C# the convention is to name methods PascalCase, even if it's an acronym.
Actual name should be:
`CalculateBmi()`